src/sinks/aws_cloudwatch_logs/request.rs

```diff
@@ -227,11 +228,9 @@ impl Client {
         let group_name = self.group_name.clone();
         let stream_name = self.stream_name.clone();
         let headers = self.headers.clone();
+
         Box::pin(async move {
-            // #12760 this is a relatively convoluted way of changing the headers of a request
-            // about to be sent. https://github.com/awslabs/aws-sdk-rust/issues/537 should
-            // eventually make this better.
```

Comment:
**note**: This wasn't strictly part of the upgrade, I just noticed the linked issue had been resolved previously and I wanted to make the world a slightly better place 😄